client/src/components/Search/index.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+import { Select, Radio, Button } from 'antd';
+import world from 'full-countries-cities';
+
+import Keywords from '../Keywords';
+
+import './style.css';
+
+const { Option } = Select;
+
+class Search extends Component {
+  state = {
+    country: undefined,
+    city: undefined,
+    lookingFor: '',
+    keyword: '',
+    viewKeywords: false,
+  };
+
+  lookingFor = { STAY: 'stay', EAT: 'eat', SHOP: 'shop' };
```

Comment:
just a minor thing, no need to tie it to the component (this will create a new `lookingFor` property for every instance of the `Search` component). Pls move it outside of the class. Also, enums are generally considered to be "types", so can use PascalCase for the name.
```js
const LookingFor = Object.freeze({ STAY: 'stay', EAT: 'eat', SHOP: 'shop' });
```
`Object.freeze()` makes sure the object can't be modified